src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/angular_template.html

```diff
@@ -0,0 +1,5 @@
+<kbn-management-app section="kibana">
+  <kbn-management-indices>
+    <div id="react"></div>
```

Comment:
It might sound paranoid but to be sure we avoid collisions can we go with a more specific `id` value, e.g. "createIndexPatternWizard"?